worker-message-prioritization-distribution/src/main/java/com/github/workerframework/workermessageprioritization/redistribution/DistributorModule.java

```diff
@@ -156,13 +162,26 @@ ConnectionFactory provideConnectionFactory(final MessageDistributorConfig messag
         ConnectionFactory connectionFactory = new ConnectionFactory();
 
         try {
+            if (messageDistributorConfig.getRabbitmqProtocol().equalsIgnoreCase("amqps")) {
+                final String tmfAlgorithm = System.getProperty(
+                        TRUST_MANAGER_FACTORY_ALGORITHM, TrustManagerFactory.getDefaultAlgorithm()
+                );
+                final TrustManagerFactory trustManagerFactory = TrustManagerFactory.getInstance(tmfAlgorithm);
+                trustManagerFactory.init((KeyStore) null);
```

Comment:
@andyreidz the `init()` method throws a `KeyStoreException`

Reply:
Sorry, my mistake, you're not using the util library from worker framework here :)